### PR TITLE
Extended DatedVehicleJourney 

### PR DESCRIPTION
**Pull request by syversenkr**
with alteration (ServiceAlterationType, ref NormalDatedVehicleJourney) and list of replacement (i.e. supplement) references (replacedJourneys)

The latter requested to maintaining a reference "backlog" for journey specific Access Rights (e.g. train X on date Y at time Z, perhaps even with allocated seat) when the original journey is replaced/supplemented (e.g. rail replacement bus)

> Moved ServiceAlterationType from NormalDatedVehicleJourneyGroup into DatedVehicleJourneyGroup as it will be used by both

### CODE REVIEW

File: xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_support.xsd [hunk 1]
```diff
@@ -95,6 +95,25 @@
     </xsd:annotation>
     <xsd:restriction base="DatedVehicleJourneyIdType"/>
   </xsd:simpleType>
+    <xsd:element name="NormalDatedVehicleJourneyRef" type="VehicleJourneyRefStructure" substitutionGroup="JourneyRef">
+    <xsd:annotation>
+      <xsd:documentation>Reference to a DATED VEHICLE JOURNEY.</xsd:documentation>
```
Comment (ue71603) [inline xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_support.xsd:100]: shouldn't it be NORMAL DATED VEHICLE JOURNEY

**Review by ue71603 — CHANGES_REQUESTED** [review]
see my single comment about the annotation. For the rest OK

File: xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_support.xsd (hunk 1 above)
Comment (syversenkr) [reply xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_support.xsd]: Corrected (was incomplete copy/paste of annotation/documentation tag), see commit https://github.com/NeTEx-CEN/NeTEx/pull/368/commits/451b61e2a0af010fd8844b43a9423b31c0daa405

**Review by ue71603 — APPROVED** [review]

**Comment by Aurige** [comment]
This PR is not editable
Closed and replaced by https://github.com/NeTEx-CEN/NeTEx/pull/411 (same content with additional decision from meeting #3 to deprecate DriverRef)

**Comment by trurlurl** [comment]
Removing "needs documentation update" label since closed without having been merged.

**Pull request closed without merging**